src/platform/configuration/config/template/template.interface.ts

```diff
@@ -1,11 +1,9 @@
 import { IOpportunityTemplate } from './opportunity.template.interface';
 import { IUserTemplate } from './user.template.interface';
-import { IOrganizationTemplate } from './organization.template.interface';
 
 export interface ITemplate {
   name: string;
   description?: string;
   users?: IUserTemplate[];
   opportunities?: IOpportunityTemplate[];
-  organizations?: IOrganizationTemplate[];
```

Comment:
How is this change related to the PR?

Reply:
The tagsets are created now by the entity holding the profile; they are no longer created by the client. This was used by the client to create tagsets - so it has to be removed.